hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptor.java

```diff
@@ -531,4 +536,107 @@ public void testGetClusterMetricsRequest() throws YarnException, IOException {
             GetClusterMetricsResponse.class);
     Assert.assertEquals(true, clusterMetrics.isEmpty());
   }
+
+  /**
+   * This test validates the correctness of
+   * GetApplicationsResponse in case the
+   * application exists in the cluster.
+   */
+  @Test
+  public void testGetApplicationsResponse()
+          throws YarnException, IOException, InterruptedException {
+    LOG.info("Test FederationClientInterceptor: " +
```

Comment:
Does this fit in 80 chars?

Reply:
Yes, no errors in checkstyle.

Reply:
I mean that putting the whole thing would fit in 80:
```
LOG.info("Test FederationClientInterceptor: Get Applications Response");
```

Reply:
Done.